frontend/src/components/forms/EditEmployee.tsx

```diff
@@ -0,0 +1,286 @@
+import React, { useEffect, useState } from "react";
+import {
+  Box,
+  Button,
+  Checkbox,
+  Divider,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  ModalFooter,
+  Radio,
+  RadioGroup,
+  Text,
+} from "@chakra-ui/react";
+import { INVITE_EMPLOYEE_ERROR } from "../../constants/ErrorMessages";
+import CreateToast from "../common/Toasts";
+import { User, UserRole } from "../../types/UserTypes";
+import UserAPIClient from "../../APIClients/UserAPIClient";
+
+const RoleOptions = [
+  UserRole.RELIEF_STAFF,
+  UserRole.ADMIN,
+  UserRole.REGULAR_STAFF,
+];
+
+type Props = {
+  employee: User;
+  isOpen: boolean;
+  toggleClose: () => void;
+};
+
+const EditEmployee = ({
```

Comment:
I think the `getUsers` method from `EmployeeDirectory` should be passed to this component. Currently changes to employee information do not get updated on the frontend until a refresh so it pulls the wrong default values if that employee is edited again.

Reply:
While I do agree, I think we're going to have to end up modifying how we do pagination anyways so leaving it as reduces the amount of code clutter we have to do in the future

Reply:
I think I'm going to modify the pagination because it brings a lot of weird extra code that we shouldn't need. If it's okay I'm going to leave it like this for now and revisit in a few days.